packages/website/src/components/Curriculum/TechSlidingAnimation.ts

```diff
@@ -0,0 +1,8 @@
+import { keyframes } from '@coderscamp/ui/components/Keyframes';
+
+const sliding = keyframes`
+   100% {
+    transform: translateX(-66.66%);
+  }
+`;
+export const animation = `${sliding} linear infinite`;
```

Comment:
💭 You import this animation only in one place so maybe there is no need to move it to a separate file